packer-variables/windows/default-args-windows.j2

```diff
@@ -0,0 +1,38 @@
+{
+  "additional_executables_destination_path": "C:\\ProgramData\\Temp",
+  "additional_executables_list": "http://{{ host_ip }}:{{ artifacts_container_port }}/artifacts/{{ kubernetes_version }}/bin/windows/amd64/registry.exe,http://{{ host_ip }}:{{ artifacts_container_port }}/artifacts/{{ kubernetes_version }}/bin/windows/amd64/goss.exe",
+  "additional_executables": "true",
+  "additional_url_images": "false",
+  "additional_url_images_list": "",
+  "additional_prepull_images": "",
+  "build_version": "{{ os_type }}-kube-{{ kubernetes_series }}-{{ ova_ts_suffix }}",
+  "cloudbase_init_url": "http://{{ host_ip }}:{{ artifacts_container_port }}/artifacts/{{ kubernetes_version }}/bin/windows/amd64/CloudbaseInitSetup_x64.msi",
+  "cloudbase_real_time_clock_utc": "true",
+  "containerd_url": "http://{{ host_ip }}:{{ artifacts_container_port }}/artifacts/{{ kubernetes_version }}/bin/windows/amd64/cri-containerd.tar",
+  "containerd_sha256_windows": "{{ containerd_sha256_windows_amd64 }}",
+  "containerd_version": "{{ containerd }}",
+  "convert_to_template": "true",
+  "create_snapshot": "false",
+  "disable_hypervisor": "false",
+  "disk_size": "40960",
+  "goss_inspect_mode": "false",
+  "kubernetes_base_url": "http://{{ host_ip }}:{{ artifacts_container_port }}/artifacts/{{ kubernetes_version }}/bin/windows/amd64",
+  "kubernetes_series": "{{ kubernetes_series }}",
+  "kubernetes_semver": "{{ kubernetes_version }}",
+  "kubernetes_typed_version": "{{ image_version }}",
+  "load_additional_components": "true",
+  "netbios_host_name_compatibility": "false",
+  "nssm_url": "http://{{ host_ip }}:{{ artifacts_container_port }}/artifacts/{{ kubernetes_version }}/bin/windows/amd64/nssm.exe",
+  "prepull": "false",
+  "pause_image": "localhost:5000/vmware.io/pause:{{ pause }}",
+  "runtime": "containerd",
+  "template": "",
+  "unattend_timezone": "Pacific Standard Time",
+  "windows_updates_categories": "",
+  "windows_updates_kbs": "",
+  "wins_url": "",
+  "custom_role": "true",
+  "custom_role_names": "/image-builder/images/capi/image/ansible-windows",
+  "ansible_user_vars": "artifacts_container_url=http://{{ host_ip }}:{{ artifacts_container_port }} imageVersion={{ image_version|replace('-', '.') }} registry_store_archive_url=http://{{ host_ip }}:{{ artifacts_container_port }}/artifacts/{{ kubernetes_version }}/registries/{{ registry_store_path }}",
+  "debug_tools": "false"
+}
```

Comment:
You set the `vmx_version: 17` in `default-args.j2` which may override the upstream image builder Windows config.

Could you also add this line for Windows in this file?
`"vmx_version": "18"`

Reply:
Can you please validate now.. updated the version for windows